### ISSUE TITLE
Turn off automatic slideshow

### ISSUE THREAD

**Issue by dennisbruinen** [cbopp-art/lightcase#48]
Hi there,

Awesome Lightbox plugin! Is it possible to add a setting option in which you can turn off the automatic sliding after clicking on an image? 

I think it would be rather easy to implement this functionality, by adding the following setting: 
```javascript
autoSlideshow: true
```

And adding an extra option to the the following if-statement:
```javascript
// Only start the timeout if slideshow is not pausing and the autoSlideshow setting is true
if (!_self.objects.nav.hasClass(_self.settings.classPrefix + 'paused') && _self.settings.autoSlideshow) {
```

Another small remark, your demo-page on http://cornel.bopp-art.com/lightcase/#demo returns an error in the original version of Internet Explorer 9, 10 and 11 (and maybe more).

Thanks in advance!

**Comment by rianneclick** [comment]
Hi,

I need the same functionality.

Can you post an update about this matter please?

Thank you in advance.

**Comment by cbopp-art** [comment]
Ok, thanks for request - Option for autostart slideshow will be realized with one of the next releases!

**Comment by cbopp-art** [comment]
Feature is implemented and already available in current development branch 2.3.x. Next official release will come quite soon.

**Comment by rianne-o** [comment]
Great! Thank you very much

**Comment by dennisbruinen** [comment]
Thanks for the update!

**Comment by rianne-o** [comment]
Hi cbopp-art,

Thanks again for the new feature. We've tested it and it works fine. 

But, in cases we don't want the option for the automatic slideshow at all. So we want to hide the play / pause button.

We can't do this by adding the style display:none; to the button since you add the style display:block; inline on the element. So we're not able to overwrite this with css and we don't want to edit the source code of lightcase.

Would you please consider implementing the functionality to disable the slideshow all together as well? This also means hiding the play button when disableSlideshow is set to true (the option name is just a suggestion).

Looking forward to hear from you.

**Comment by cbopp-art** [comment]
Hi!

Just Make sure that the option slideshow is set to false for those links.

Regards,
Cornel

**Comment by rianne-o** [comment]
We already tried this, but nothing happens. The play button stays visible.

Added this option to the initialization: `slideshow : false` 

Just for clarity, we do want to be able to click / swipe through the group of image. We only want to disable the play / pause functionality.

Is it possible that this is a bug?

**Comment by cbopp-art** [comment]
How actually do you initialize Lightcase? Could you show me that snippet?

**Comment by rianne-o** [comment]
```
$(document).ready(function() {
    $('.js-lightcase-slideshow').lightcase({
        transition: 'scrollHorizontal',
        fullscreenModeForMobile: true,
        autoSlideshow: false,
        slideshow: false
    });
});
```

**Comment by cbopp-art** [comment]
And what is the value of your data.rel attribute in html tags?

**Comment by cbopp-art** [comment]
And what is the value of your "data-rel" attribute in html tags?

**Comment by rianne-o** [comment]
```
data-rel="slideshowCollection:slideshow" 
class="js-lightcase-slideshow"
```

Also tried to remove `slideshow` from the `data-rel`, but that disables clicking through the group of image, which is not what we want.

**Comment by cbopp-art** [comment]
Have a look at the docs how to use data-rel for collections as well
as slideshow
and what is different.

use something like:
lightcase:myCollection

end with
:slideshow is reserved.

**Comment by rianne-o** [comment]
Thank you, done that. Found the following in the documentation:

**Comment by rianne-o** [comment]
Thanks for your help so far. I've taken another look at the documentation and found the following:

"Collections
Couple of links with same attribute will open all of them as a collection."
`<a href="path/to/media1.jpg" data-rel="lightcase:myCollection">Your link description or thumb</a>`

"Slideshow
Collections with trailing ":slideshow" in the attribute 'data-rel' will open them as a slideshow."
`<a href="path/to/media.jpg" data-rel="lightcase:myCollection:slideshow" title="Your title">Your link description or thumb</a>`

"slideshow	Boolean     false	Default slideshow initialization for groups."

So, changed my code to `<a href="<image>" class="js-lightcase-slideshow" data-rel="lightcase:myCollection:slideshow">`

and

```
$(document).ready(function() {
    $('.js-lightcase-slideshow').lightcase({
        transition: 'scrollHorizontal',
        fullscreenModeForMobile: true,
        autoSlideshow: false,  
        slideshow: false
    });
});
```

But the play button is still visible. See attached screenshot.

![scr_lightcase](https://cloud.githubusercontent.com/assets/6838453/12613546/c72b0872-c4fa-11e5-8d3b-492439027e02.jpg)

**Comment by cbopp-art** [comment]
slideshow means “autoplay”. open just as collection (group), it will let you browse through the collection via next/prev buttons only.

change to this:
 <a href="<image>" class="js-lightcase-slideshow" data-rel=“lightcase:myCollection">

**Comment by cbopp-art** [comment]
just seen:

option autoSlideshow does not exist. Is actually called “slideshowAutoStart"

**Comment by rianne-o** [comment]
> Change to this:
> <a href="<image>" class="js-lightcase-slideshow" data-rel=“lightcase:myCollection">

That did the trick!

I already tried to remove `slideshow` from the `data-rel` attr before, but that didn't make any difference. 
Found out that this was because my collection name was 'slideshowCollection'. 
It looks like as soon as the term 'slideshow' is present within the `data-rel` (even though it's in the collection name), the automatic slideshow functionality fires.

Thanks a lot for your help! Great plugin!
